extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/sources/generated/RasterDemSourceTest.kt

```diff
@@ -243,6 +243,50 @@ class RasterDemSourceTest : BaseStyleTest() {
     assertEquals(1L, testSource.maxOverscaleFactorForParentTiles)
   }
 
+  @Test
+  @UiThreadTest
+  fun tileRequestsDelayTest() {
+    val testSource = rasterDemSource(SOURCE_ID) {
```

Comment:
What's the default value for the delay? Should we add a test case for the default value?

Reply:
default is no delay(equal to 0), but there's also no default value in the spec, otherwise we would generate the default value test.